packages/next-starter/src/modules/core/components/Page.tsx

```diff
@@ -42,7 +42,7 @@ export const Page = (props: IWrapperPage.IProps) => {
     <Spin tip="Loading..." size="large" spinning={loading}>
       <ThemeProvider theme={theme}>
         <Container className={`${weakColor ? 'weakColor' : ''} ${boxed ? 'boxed shadow-sm' : ''}`}>
-          {!isNotDashboard && <Header {...props} />}
+          {!isNotDashboard && <Header HeaderMainSection={HeaderMainSection} />}
```

Comment:
Note that this may introduce breaking change? oh...nvm, we were not using props from Header anyway.